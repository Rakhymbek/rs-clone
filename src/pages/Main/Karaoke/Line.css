.WrapperStyle {
  position: relative;
  display: inline-block;
}

.FontStyle {
  font-size: 2.5vw;
  color: white;
  text-shadow: -2px 0 black, 0 2px black, 2px 0 black, 0 -2px black;
}

.ActiveStyle {
  font-size: 2.5vw;
  color: white;
  text-shadow: -2px 0 black, 0 2px black, 2px 0 black, 0 -2px black;
  position: absolute;
  left: 0;
  top: 0;
  color: blue;
  z-index: 1;
  text-shadow: -2px 0 white, 0 2px white, 2px 0 white, 0 -2px white;
}
