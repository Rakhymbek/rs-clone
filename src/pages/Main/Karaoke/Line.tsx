import React, { FC } from 'react';
import './Line.css'

interface ILineProps {
    text: string;
}

const Line: FC<ILineProps> = ({text}) => {
        console.log(text);
       
        return (
          <div className='WrapperStyle'>
            <div className='FontStyle'>{text}</div>
            <div className='ActiveStyle'>{text}</div>
          </div>
        );
};

export default Line;