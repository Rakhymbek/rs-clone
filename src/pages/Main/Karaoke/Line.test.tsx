import { render } from '@testing-library/react';
import Line from './Line';

describe('<Line />', () => {
  it('should render successfully and match snapshot', () => {
    const { container } = render(<Line text="Song text"></Line>);

    expect(container).toMatchSnapshot();
  });
});
