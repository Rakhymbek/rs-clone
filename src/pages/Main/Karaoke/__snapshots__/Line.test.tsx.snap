// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Line /> should render successfully and match snapshot 1`] = `
<div>
  <div
    class="WrapperStyle"
  >
    <div
      class="FontStyle"
    >
      Song text
    </div>
    <div
      class="ActiveStyle"
    >
      Song text
    </div>
  </div>
</div>
`;
